src/utils/attendantFormInputValidations.js

```diff
@@ -0,0 +1,38 @@
+const validateNameInput = (value) => {
+  const nameRegex = /^[A-Za-zÀ-ÖØ-öø-ÿ'\\-]{2,50}$/;
+
+  return !!value.match(nameRegex);
+};
+
+const validateLastNameInput = (value) => {
+  const lastNameRegex = /^[A-Za-zÀ-ÖØ-öø-ÿ'\\-]{2,50}$/;
+
+  return !!value.match(lastNameRegex);
+};
+
+const validateJobTitleSelection = (value) => {
+  return value !== "";
+};
+
+const validateAgeInput = (value) => {
+  const ageRegex = /^(1[5-9]|[2-9][0-9]|1[01][0-9]|120)$/;
+
+  return !!value.match(ageRegex);
+}
+
+const validateInputs = ({name, value}) => {
```

Comment:
This function can be exported instead of the method used below
```suggestion
export const validateInputs = ({name, value}) => {
```